cukinia/common_security_tests.d/auditd.conf

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+cukinia_log "$(_colorize yellow "--- check auditd configuration ---")"
+modules_function=" \
+    insmod \
+    kmod \
+    modprobe \
+    rmmod \
+"
+
+syscall_list=" \
+    ioperm \
+    prctl \
+    ptrace \
+"
+
+file_monitoring="\
+    open \
+    openat \
+    unlink \
+"
+
+as "ANSSI-BP-028-R73 - auditd service is active" \
+    cukinia_systemd_unit auditd
+
+as "ANSSI-BP-028-R73 - auditd is configure to output in syslog" cukinia_cmd \
+    grep -q "active = yes" /etc/audit/plugins.d/syslog.conf
+
+
+for function in ${modules_function}; do
+    /sbin/$function &>/dev/null
+    as "ANSSI-BP-028-R73 - $function call is log" cukinia_cmd \
+        grep -q $function /var/log/audit/audit.log
+done
+
+touch /etc/test
+as "ANSSI-BP-028-R73 - modification in /etc/ is log" cukinia_cmd \
+    grep -q "/etc/test" /var/log/audit/audit.log
+
+mount /dev/loop4 /mnt/ &>/dev/null
+as "ANSSI-BP-028-R73 - mount/umount call is log" cukinia_cmd \
+    grep -q "mount" /var/log/audit/audit.log
+
+for syscall in ${syscall_list}; do
+    python3 /usr/bin/$syscall".py" &>/dev/null
```

Comment:
"/usr/bin/${syscall}.py"

Reply:
Why `&`? It's ok if you do it in two times to avoid timing problems:
```
for syscall in ${syscall_list}; do
    python3 /usr/bin/$syscall".py" &>/dev/null
done
for syscall in ${syscall_list}; do
 as BLABLA cukinia_cmd
    grep -q $syscall /var/log/audit/audit.log
done
```

Reply:
I did that to be sure that print output is not shown when running Cukinia test (needed for `mount` for example). But in this case, it is indeed to needed.